web/less/room.less

```diff
@@ -1178,27 +1206,23 @@ footer.main {
 }
 
 .room-name {
-  font-size: 16px;
+  font-size: 13px;
+  margin: 6px 0 20px;
   word-break: break-word;
   display: block;
   text-overflow: ellipsis;
   overflow: hidden;
-  color: #fff;
-  line-height: 18px;
 }
 
 #enter.btn {
+  color: #fff;
```

Comment:
I think we should be using the variables in variables.less to define colors and use them. Leaving that aside we have hex values all over the place in less files so this would be a task on its own. If you agree we can create a tech debt ticket.

Reply:
Yes, let's do this as a tech debt task.